### PR TITLE
Updated idlejitter-plugin docs

### PR DESCRIPTION
**Pull request by kickoke**
Because the documentation for the idlejitter.plugin didn't match with the structure of our plugins and collectors, I made some refinements.

### CODE REVIEW

File: collectors/idlejitter.plugin/README.md [hunk 1]
```diff
@@ -5,16 +5,23 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/idlej
 
 # idlejitter.plugin
 
-It works like this:
+Idle jitter is the delay of a CPU process.
```
Comment (ilyam8) [inline collectors/idlejitter.plugin/README.md:8]: Not an expert, my understanding of this plugin - it tries to measure CPU preemption latency (task scheduling). @netdata/agent, back me up on this.

File: collectors/idlejitter.plugin/README.md [hunk 2]
```diff
@@ -5,16 +5,23 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/idlej
...
 
+## How Netdata measures idle jitter
 A thread is spawned that requests to sleep for 20000 microseconds (20ms).
```
Comment (ilyam8) [inline collectors/idlejitter.plugin/README.md:11]:
```suggestion
## How Netdata measures idle jitter

A thread is spawned that requests to sleep for 20000 microseconds (20ms).
```

File: collectors/idlejitter.plugin/README.md [hunk 3]
```diff
@@ -5,16 +5,23 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/idlej
...
 
 This number is useful:
 
-1.  in real-time environments, when the CPU jitter can affect the quality of the service (like VoIP media gateways).
-2.  in cloud infrastructure, at can pause the VM or container for a small duration to perform operations at the host. 
+- In real-time environments when the CPU jitter can affect the quality of the service (like VoIP media gateways).
+- In cloud infrastructure that can pause the VM or container for a small duration to perform operations at the host.
+
+## Charts
+idlejitter.plugin generates the idlejitter chart which measures CPU idle jitter in milliseconds lost per second.
```
Comment (ilyam8) [inline collectors/idlejitter.plugin/README.md:22]:
```suggestion
## Charts

idlejitter.plugin generates the idlejitter chart which measures CPU idle jitter in milliseconds lost per second. 
```

File: collectors/idlejitter.plugin/README.md [hunk 4]
```diff
@@ -5,16 +5,23 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/idlej
...
-1.  in real-time environments, when the CPU jitter can affect the quality of the service (like VoIP media gateways).
-2.  in cloud infrastructure, at can pause the VM or container for a small duration to perform operations at the host. 
+- In real-time environments when the CPU jitter can affect the quality of the service (like VoIP media gateways).
+- In cloud infrastructure that can pause the VM or container for a small duration to perform operations at the host.
+
+## Charts
+idlejitter.plugin generates the idlejitter chart which measures CPU idle jitter in milliseconds lost per second. 
+
+## Configuration
+This chart is available without any configuration.
```
Comment (ilyam8) [inline collectors/idlejitter.plugin/README.md:25]:
```suggestion
## Configuration

This chart is available without any configuration. 
```

File: collectors/idlejitter.plugin/README.md (hunk 1 above)
Comment (Ferroin) [reply collectors/idlejitter.plugin/README.md:8]: Both are kind of correct.

The internal measurement is making a call to sleep (really nanosleep or uslep, depending on the system), and then tracking how much longer it takes for the process to be woken up than the time it asked to sleep for. This is functionally ‘lost’ time from the  perspective of the process in question, and high values cause all kinds of problems for realtime and near-realtime applications.

It’s technically a function of preemption latency, and it is also a measure of (one type of) induced delay in processes utilizing such timing functions

I would suggest phrasing it as:

```suggestion
Idle jitter is a measure of delays in timing for user processes caused by scheduling limitations.
```

File: collectors/idlejitter.plugin/README.md [hunk 5]
```diff
@@ -5,16 +5,23 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/idlej
...
 When the system wakes it up, it measures how many microseconds have passed.
 The difference between the requested and the actual duration of the sleep, is the idle jitter.
 This is done at most 50 times per second, to ensure we have a good average. 
 
 This number is useful:
 
-1.  in real-time environments, when the CPU jitter can affect the quality of the service (like VoIP media gateways).
-2.  in cloud infrastructure, at can pause the VM or container for a small duration to perform operations at the host. 
+- In real-time environments when the CPU jitter can affect the quality of the service (like VoIP media gateways).
+- In cloud infrastructure that can pause the VM or container for a small duration to perform operations at the host.
```
Comment (Ferroin) [inline collectors/idlejitter.plugin/README.md:19]:
```suggestion
- In multimedia-streaming environments such as VoIP gateways, where the CPU jitter can affect the quality of the service.
- On time servers and other systems that require very precise timing, where CPU jitter can actively interfere with timing precision.
- On gaming systems, where CPU jitter can cause frame drops and stuttering.
- In cloud infrastructure that can pause the VM or container for a small duration to perform operations at the host.
```

**Comment by kickoke** [comment]
Thank you for your feedback @ilyam8 @Ferroin 🤗 
I think the PR is now ready to be merged.

**Review by ilyam8 — APPROVED** [review]

**Review by Ferroin — APPROVED** [review]

**Comment by Ferroin** [comment]
Merging as the remaining CI checks are irrelevant to the changes.

**Pull request merged**